spec/services/bulk_url_uploader/results_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+describe BulkUrlUploader::Results do
+  subject(:results) { described_class.new('test bulk uploader') }
+
+  describe '#errors' do
+    context 'when there are "url not found" errors' do
```

Comment:
This context should be `when there are "Url has already been taken" errors`

Reply:
Changed